### PR TITLE
Fix kbd description in gdextension docs system.

### PR DESCRIPTION
**Pull request by Ivorforce**
Referring to https://github.com/godotengine/godot-docs/edit/master/tutorials/scripting/gdscript/gdscript_documentation_comments.rst

### CODE REVIEW

**Comment by dalexeev** [comment]
Could you please remove the following line as well? I think it's because of copy-paste from [`RichTextLabel` tutorial](https://docs.godotengine.org/en/latest/tutorials/ui/bbcode_in_richtextlabel.html#reference). You don't need to handle `meta_clicked` in Editor Help.

https://github.com/godotengine/godot-docs/blob/da350f27139a52a975590d617960a3452521df51/tutorials/scripting/gdextension/gdextension_docs_system.rst?plain=1#L147

Note that Editor Help BBCodes are currently documented 3 times:

* [GDScript documentation comments](https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_documentation_comments.html#bbcode-and-class-reference)
* [GDExtension documentation system](https://docs.godotengine.org/en/latest/tutorials/scripting/gdextension/gdextension_docs_system.html#documentation-styling)
* [Class reference primer](https://docs.godotengine.org/en/latest/contributing/documentation/class_reference_primer.html#doc-class-reference-bbcode)

File: tutorials/scripting/gdextension/gdextension_docs_system.rst [hunk 1]
```diff
@@ -116,9 +116,9 @@ Currently they supported tags for the GDExtension documentation system are:
     - ``[s]{text}[/s]``
 
   * - | **kbd**
-      | Makes ``{text}`` use the mono font and styles the text color and background like a shortcut.
+      | Indicates ``{text}`` is a keyboard shortcut.
```
Comment (dalexeev) [inline tutorials/scripting/gdextension/gdextension_docs_system.rst:119]: Is it possible to keep "Makes..." for consistency?

Comment (Ivorforce) [reply tutorials/scripting/gdextension/gdextension_docs_system.rst:119]: "Makes text use a grey beveled background, indicating a keyboard shortcut."?

**Comment by Ivorforce** [comment]
> Could you please remove the following line as well? I think it's because of copy-paste from [`RichTextLabel` tutorial](https://docs.godotengine.org/en/latest/tutorials/ui/bbcode_in_richtextlabel.html#reference). You don't need to handle `meta_clicked` in Editor Help.

Done

> Note that Editor Help BBCodes are currently documented 3 times:
> 
> * [GDScript documentation comments](https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_documentation_comments.html#bbcode-and-class-reference)
> * [GDExtension documentation system](https://docs.godotengine.org/en/latest/tutorials/scripting/gdextension/gdextension_docs_system.html#documentation-styling)
> * [Class reference primer](https://docs.godotengine.org/en/latest/contributing/documentation/class_reference_primer.html#doc-class-reference-bbcode)

Yeah, it would be good to de-duplicate this info in a future PR.

File: tutorials/scripting/gdextension/gdextension_docs_system.rst (hunk 1 above)
Comment (Ivorforce) [reply tutorials/scripting/gdextension/gdextension_docs_system.rst]: I went ahead with that, feel free to resolve if that sounds better to you.

**Review by paddy-exe — APPROVED** [review]

**Comment by paddy-exe** [comment]
Needs a squash though @Ivorforce

**Comment by Ivorforce** [comment]
Isn't that possible from the GUI?
image

**Comment by paddy-exe** [comment]
@Ivorforce While it is technically possible it pollutes the commit history. Please read the appropriate section for the contributor documentation: https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase

**Comment by Ivorforce** [comment]
An upside of the squash-merge is that the tests don't have to run again :)

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you! Merged.

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.3 in https://github.com/godotengine/godot-docs/pull/10038.

**Pull request merged**